test-e2e/apps/waits.html

```diff
@@ -75,6 +75,11 @@
         waitCollection++;
         if (waitCollection > 10) clearInterval(waitCollectionInterval);
     }, 200);
+
+    setTimeout(() => {
+        window.document.title = 'title changed';
+        window.document.location.href = '#anchor';
+    }, 1234);
```

Comment:
is 1234 some like dummy data?

Reply:
yep, simulating async update of browser url and title